monitor/utils/monasca/connector.py

```diff
@@ -91,8 +85,8 @@ def send_metrics(self, measurements):
         try:
             monasca_client = self._get_monasca_client()
             monasca_client.metrics.create(**batch_metrics)
-        
+
         except exc.HTTPException as httpex:
-            print httpex.message
```

Comment:
What is the main purpose of this modification? Can you please document this refactor in the description of this PR? This issue repeats itself through all file.